headers/augment_headers.py

```diff
@@ -203,6 +203,9 @@ def add_aliases(self) -> int:
                     if ";" in line.split("//", 1)[0]:
                         # Symbol declaration is finished; add aliases
                         for alias in aliases:
+                            if mark_as_deprecated:
+                                f.write(f"__attribute((deprecated(\"Renamed to '{aliased_symbol}'\"))) ")
```

Comment:
I'm not entirely sure why `__attribute` works, but I believe the official syntax has both leading and trailing underscores, i.e. `__attribute__((deprecated(...)))`